descqa/apparent_mag_func_test.py

```diff
@@ -133,25 +150,37 @@ def run_on_single_catalog(self, catalog_instance, catalog_name, output_dir):
         if not mag_field_key:
             return TestResult(skipped=True, summary='Catalog is missing requested quantity: {}'.format(self.possible_mag_fields))
 
+        # check to see if catalog is a light cone
+        # this is required since we must be able to calculate the angular area
+        try:
+            if not catalog_instance.lightcone:
+                return TestResult(skipped=True, summary="Catalog is not a light cone.")
+        except AttributeError:
+            if not catalog_instance.has_quantity('ra') or not catalog_instance.has_quantity('dec'):
+                return TestResult(skipped=True, summary="'ra' and/or 'dec' not available to compute sky area")
+            
+        # check to see the angular area if an attribute of the catalog
+        try:
+            sky_area = catalog_instance.sky_area
+        except AttributeError:
+            try:
+                sky_area = get_sky_area(catalog_instance)
+            except:
+                return TestResult(skipped=True, summary="'sky_area' cannot be determined from catalog")
```

Comment:
Provide error information
```suggestion
                return TestResult(skipped=True, summary="'sky_area' cannot be determined from catalog: {}".format(e))
```